### ISSUE TITLE
Organization?

### ISSUE THREAD

**Issue by robotorigami** [Birch-san/juicysfplugin#29]
juicy plays the sounds great and I love having an envelope for each sound, but I'm wondering if this feature would be possible: I've been using a lot of video game soundfonts, most of which are giant sf2 files which load in a bunch of sounds, and sometimes the sounds are just numbered. The ability to rename sounds within each sf2 or delete certain ones would be so convenient. Just an idea.

**Comment by Birch-san** [comment]
I agree it's a problem, but it feels like the better place to fix this is in the soundfont itself? I'd recommend finding a tool that lets you modify the soundfont, and rename the presets in it.
